src/main/java/mate/project/mapper/UserMapper.java

```diff
@@ -0,0 +1,16 @@
+package mate.project.mapper;
+
+import mate.project.config.MapperConfig;
+import mate.project.dto.UserRegistrationRequestDto;
+import mate.project.dto.UserResponseDto;
+import mate.project.model.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface UserMapper {
+    UserResponseDto toUserResponseDto(User user);
+
+    @Mapping(target = "id", ignore = true)
+    User toModel(UserRegistrationRequestDto userRegistrationRequestDto);
```

Comment:
The @Mapping annotation is used to ignore the 'id' field when mapping from UserRegistrationRequestDto to User. Given that UserRegistrationRequestDto likely doesn't have an 'id' field, this annotation may not be necessary.

Reply:
Without having this @Mapping annotation here maven is keep throwing me issues that field id is not mapped